cmd/main.go

```diff
@@ -86,7 +84,7 @@ func main() {
 	flag.StringVar(&storePath, "store-path", ".", "The path to store publisher and subscription info.")
 	flag.StringVar(&transportHost, "transport-host", "http://ptp-event-publisher-service-NODE_NAME.openshift-ptp.svc.cluster.local:9043", "The transport bus hostname or service name.")
```

Comment:
transportHost should be renamed apiHost

Reply:
On second thought, maybe it is best to keep the naming of these flags for now, since changing them would require migration path and lots of documentation updates.
I think for now we can use transport-host to get both apiHost and apiPort (9043), keep the api-port flag as is but not actually use its value.

Reply:
We can change the name later in different PRs since it would require changes in ptp-operator and test scripts.